### PR TITLE
Use glob for signing resources

### PR DESCRIPTION
**Pull request by jaredpar**

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @nguerrera

**Review by nguerrera — APPROVED** [review]

File: src/Dependencies/SignTool/SignTool.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="RoslynTools.Microsoft.SignTool" Version="0.2.0-beta" />
+    <PackageReference Include="RoslynTools.Microsoft.SignTool" Version="0.3.3-beta" />
```
Comment (nguerrera) [inline src/Dependencies/SignTool/SignTool.csproj]: 3.3.0 ?

Comment (nguerrera) [inline src/Dependencies/SignTool/SignTool.csproj]: or vice versa?

**Review by nguerrera — CHANGES_REQUESTED** [review]
waiting on version mismatch

**Comment by nguerrera** [comment]
Unfortunately, our CI doesn't validate signing like other repos yet. Since this is going to master (2.1), I can safely merge and check on the build once the version issue is fixed.

File: src/Dependencies/SignTool/SignTool.csproj (hunk 1 above)
Comment (jaredpar) [inline src/Dependencies/SignTool/SignTool.csproj]: That's the correct version.  See https://dotnet.myget.org/feed/roslyn-tools/package/nuget/RoslynTools.Microsoft.SignTool/0.3.3-beta

File: build/Signing/Microsoft.NET.Signing.proj [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
 
   <PropertyGroup>
-    <SignTool>$(NuGet_Packages)\roslyntools.microsoft.signtool\0.2.0-beta\tools\SignTool.exe</SignTool>
+    <SignTool>$(NuGet_Packages)\roslyntools.microsoft.signtool\3.3.0-beta\tools\SignTool.exe</SignTool>
```
Comment (nguerrera) [inline build/Signing/Microsoft.NET.Signing.proj]: Then this one is wrong.

File: src/Dependencies/SignTool/SignTool.csproj (hunk 1 above)
Comment (nguerrera) [inline src/Dependencies/SignTool/SignTool.csproj]: https://github.com/dotnet/sdk/pull/1368/files#r123643568

File: build/Signing/Microsoft.NET.Signing.proj (hunk 2 above)
Comment (jaredpar) [inline build/Signing/Microsoft.NET.Signing.proj]: I have all the right numbers, they're just in the wrong order 😄

Comment (jaredpar) [inline build/Signing/Microsoft.NET.Signing.proj]: Fixed now.

Comment (nguerrera) [inline build/Signing/Microsoft.NET.Signing.proj]: File a bug on nuget?

**Review by nguerrera — APPROVED** [review]

**Pull request merged**